.github/workflows/tests.yml

```diff
@@ -192,7 +192,6 @@ jobs:
               echo 'PySide2 is available' ||
               echo 'PySide2 is not available'
           fi
-          # Qt6 crashes on Github's ubuntu 18.04 runner.
           if [[ "${{ matrix.os }}" = ubuntu-20.04 ]]; then
```

Comment:
This `if` can be removed?

Reply:
One could think so, but if I do not miss anything it would mean that it is also installed on OSX, which changes the current behaviour if nothing else. (The comment was probably not fully correct/complete.)

Reply:
That's true; I'm wondering it maybe it'd work on macOS now? But maybe not for this PR.